quality_control_oca/models/qc_inspection.py

```diff
@@ -43,12 +45,14 @@ def _compute_product_id(self):
         copy=False,
     )
     date = fields.Datetime(
+        string="Plan Date",
         required=True,
         readonly=True,
         copy=False,
         default=fields.Datetime.now,
         states={"draft": [("readonly", False)]},
     )
+    date_done = fields.Datetime(readonly=True)
```

Comment:
Just to make the name presentation somewhat consistent among date fields.
```suggestion
    date_done = fields.Datetime(readonly=True, "Completion Date")
    create_date = fields.Datetime("Create Date")
```